crypto/Pos/Crypto/Signing/Redeem.hs

```diff
@@ -46,7 +46,7 @@ redeemDeterministicKeyGen seed =
 
 -- | Encode something with 'Binary' and sign it.
 redeemSign ::
-       (HasCryptoConfiguration, Bi a)
+       (HasCryptoConfiguration, Bi a, Bi ProtocolMagic)
```

Comment:
Why can't you import `Pos.Crypto.Binary` here?

Reply:
I think it would lead to an import cycle, but I haven't checked.

Reply:
It wouldn't :)